src/renderer/src/routes/pages/Send/index.tsx

```diff
@@ -14,9 +17,46 @@ import { TokenBalanceList } from './TokenBalanceList'
 import { TotalFee } from './TotalFee'
 
 export const SendPage = () => {
+  enum SendPageStep {
+    SelectAccount = 1,
+    SelectToken = 2,
+    SelectAmount = 3,
+    SelectContact = 4,
+  }
+
   const { t } = useTranslation('pages', { keyPrefix: 'send' })
   const { accounts } = useAccountsSelector()
-  const balanceExchange = useBalancesAndExchange(accounts)
+  const balanceExchange = useBalancesAndExchange(accounts[0])
+  const [selectedAccount, setSelectedAccount] = useState<IAccountState>()
+  const [selectedToken, setSelectedToken] = useState<TokenBalance | null>()
+  const [selectedAmount, setSelectedAmount] = useState<number>(0)
+  const [currentStep, setCurrentStep] = useState<number>(SendPageStep.SelectAccount)
```

Comment:
Use the enum to type this state
```suggestion
  const [currentStep, setCurrentStep] = useState<SendPageStep>(SendPageStep.SelectAccount)
```